language-support/ts/codegen/src/TsCodeGenMain.hs

```diff
@@ -51,13 +52,14 @@ configConsts :: T.Text -> ConfigConsts
 configConsts sdkVersion = ConfigConsts
   { pkgDependencies = HMS.fromList
       [ (NpmPackageName "@mojotech/json-type-validation", NpmPackageVersion "^3.1.0")
-      , (NpmPackageName "@daml/types", NpmPackageVersion sdkVersion)
+      , (NpmPackageName "@daml/types"
```

Comment:
We could do something with the resolutions field in the daml2ts package.json if that's considered preferable.

Reply:
Yes, I'd be more happy if `daml2ts` wouldn't know about this and the actual tests do the "rerouting". Doesn't https://github.com/digital-asset/daml/blob/master/language-support/ts/codegen/tests/ts/package.json do this already anyways?